modular_skyrat/master_files/code/modules/clothing/shoes/antigravboots.dm

```diff
@@ -0,0 +1,32 @@
+///This file houses the antigravity boots item.
+/obj/item/clothing/shoes/antigrav_boots
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/shoes.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
+	desc = "The QM's 'special' boots. Control Click to toggle anti-gravity functions."
+	name = "anti-gravity boots"
+	icon_state = "walkboots" //Haha funny reused sprite
+	var/enabled_antigravity = FALSE
+
+/obj/item/clothing/shoes/antigrav_boots/equipped(mob/user, slot)
+	. = ..()
+	if(!slot == ITEM_SLOT_FEET)
+		return
+	if(enabled_antigravity)
+		user.AddElement(/datum/element/forced_gravity, 0)
+
+/obj/item/clothing/shoes/antigrav_boots/dropped(mob/user)
+	. = ..()
+	user.RemoveElement(/datum/element/forced_gravity, 0)
+
+/obj/item/clothing/shoes/antigrav_boots/CtrlClick(mob/living/user)
```

Comment:
could move this to right click?

Reply:
They already have to remove the boots to change it, might as well make it clankier.